src/RefetchProvider.tsx

```diff
@@ -31,16 +32,16 @@ const RefetchProvider: React.FunctionComponent<{}> = ({ children }) => {
     subscriptions.current = {}
   }, [])
 
-  const refetch = React.useCallback((category: string) => {
+  const refetch = React.useCallback((category: keyof Subscriptions) => {
     try {
       return Promise.all(
-        Object.values(
-          subscriptions.current[category]
-        ).map((subscriptonRefetch) => subscriptonRefetch())
+        Object.values(subscriptions.current[category]).map(
+          (subscriptonRefetch) => subscriptonRefetch()
+        )
       )
     } catch (e) {
       if (process.env.NODE_ENV === 'dev') {
-        console.warn(`${category} is not registered in refetch subscriptions`) // eslint-disable-line
+        console.warn(`${category.toString()} is not registered in refetch subscriptions`) // eslint-disable-line
```

Comment:
```suggestion
        console.warn(`${category} is not registered in refetch subscriptions`) // eslint-disable-line
```

Reply:
TypeScript was complaining about implicit conversion of `symbol`.